### PR TITLE
feat(docs-site): update network config, list cooldown, proving windows and bonds for every tier, remove banner

### PR DESCRIPTION
**Pull request by RogerLamTd**
@dantaik values are based on what I found in main branch TierProviderBase.sol, please let me know if any changes are necessary.

### CODE REVIEW

File: packages/docs-site/src/content/docs/network-reference/network-configuration.md [hunk 1]
```diff
@@ -3,9 +3,30 @@ title: Network configuration
...
+
+You can find all the corresponding values below in the above link too!
+
+| Tier              | Cooldown Window | Proving Window | Bond      |
+| ----------------- | --------------- | -------------- | --------- |
+| Optimistic        | 1440 minutes    | 15 minutes     | 75 TAIKO  |
+| SGX               | 240 minutes     | 60 minutes     | 150 TAIKO |
+| ZK                | 240 minutes     | 180 minutes    | 225 TAIKO |
+| SGX & ZK          | 240 minutes     | 180 minutes    | 225 TAIKO |
+| Guardian Minority | 240 minutes     | 2880 minutes   | 225 TAIKO |
```
Comment (davidtaikocha) [inline packages/docs-site/src/content/docs/network-reference/network-configuration.md:24]: seems proving window for gp in code is just `GRACE_PERIOD`?

Comment (dantaik) [reply packages/docs-site/src/content/docs/network-reference/network-configuration.md:24]: @RogerLamTd please do not use the code on the main branch, you should fine the deployed contract URL and use the verified code as reference.

Comment (dantaik) [reply packages/docs-site/src/content/docs/network-reference/network-configuration.md:24]: grace period is also removed.

Comment (RogerLamTd) [reply packages/docs-site/src/content/docs/network-reference/network-configuration.md:24]: @dantaik I see grace period has yet to be removed on the live contract, we're leaving it for a future upgrade if I'm not mistaken?

File: packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md [hunk 2]
```diff
@@ -33,7 +34,7 @@ description: Network reference page describing various important addresses on Ta
 | P256Verifier                    | `0x11A9ebA17EbF92b40fcf9a640Ebbc47Db6fBeab0` | N/A                        |
 | SigVerifyLib                    | `0x47bB416ee947fE4a4b655011aF7d6E3A1B80E6e9` | N/A                        |
 | TokenUnlock                     | `0x035AFfC82612de31E9Db2259B9482D0Dd53B7819` | N/A                        |
-| ProverSet                       | `0x34f2B21107AfE3584949c184A1E6236FFDAC4f6F` | N/A                        |
+| ProverSet                       | `0x518845daA8870bE2C59E49620Fc262AD48953C9a` | N/A                        |
```
Comment (davidtaikocha) [inline packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md:37]:
```suggestion
| ProverSet                       | `0x3022Ed0346CCE0c08268c8ad081458AfD95E8763` | N/A                        |
```

Let's use this one, address manager has not been updated yet.

File: packages/docs-site/src/content/docs/network-reference/network-configuration.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| Optimistic        | 1440 minutes    | 255 minutes    | 75 TAIKO  |
+| SGX               | 240 minutes     | 300 minutes    | 150 TAIKO |
+| ZK                | 240 minutes     | 420 minutes    | 225 TAIKO |
+| SGX & ZK          | 240 minutes     | 420 minutes    | 225 TAIKO |
+| Guardian Minority | 240 minutes     | 240 minutes    | 225 TAIKO |
+| Guardian          | 240 minutes     | 240 minutes    | 0 TAIKO   |
+
+## Tier configuration (Mainnet)
+
+You can view the full `MainnetTierRouter` tier configuration [here](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/layer1/mainnet/rollup/MainnetTierRouter.sol):
```
Comment (davidtaikocha) [inline packages/docs-site/src/content/docs/network-reference/network-configuration.mdx:29]: Let's use this link? https://etherscan.io/address/0x8f1C1D58C858e9a9eeCc587d7D51AECfd16b5542#code , code in github link will be updated after merging another PR, but we won't deploy it these days.

**Review by dantaik — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]
